getting-started/publish-article.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Publish & Share Your Article
+short_title: Publish & Share
+---
+
+
+# Publish & Share
+
+**Goal:** Publish your article or site so it’s live on the web.  
+
+
+
+## 1. Make Sure Your Project Is Ready
+
+Before publishing, confirm that your project has:
+
+- At least one article with content.
+- A valid `curvenote.yml` configuration file.
+- Any required assets (figures, datasets, notebooks) included in the project.
+
+If you’re not sure your site is configured, see:  
+[Set Up a Site →](../getting-started/set-up-site.md)
+
+
+
+## 2. Publish from the CLI
+
+Run the publish command from your project folder:
+
+```bash
+curvenote publish
```

Comment:
This command does not exist. There are currently two options which may be considered "publishing"

1. `curvenote deploy` - this makes your project live on your `.curve.space` domain, configured under `site.domains` in your `curvenote.yml`. This is the simplest way to put your content on the web.
2. `curvenote submit <site>` - submits your project to a site, e.g. lab site or journal. In this case, it is still "unpublished" - only visible to those who have admin access to the site. If you have admin access, you can also publish from the CLI with `curvenote submission publish <site>` - this will make a project available online.